### ISSUE TITLE
 Toggle: High contrast mode issue

### ISSUE THREAD

**Issue by atneik** [OfficeDev/office-ui-fabric-react#110]
On and Off states are not distinguishable in high contrast mode (black and white)

<img width="207" alt="screen shot 2016-08-09 at 2 29 46 pm" src="https://cloud.githubusercontent.com/assets/785361/17538818/d3e0eee0-5e5d-11e6-847f-e50ebd2b1e5a.png">

Should be as it is in Fabric-JS:

<img width="272" alt="screen shot 2016-08-10 at 11 09 07 am" src="https://cloud.githubusercontent.com/assets/785361/17565131/ef95f4f4-5eea-11e6-87fd-55702c7fed64.png">

**Comment by atneik** [comment]
Fixed: PR #131
